src/Compilers/Server/VBCSCompilerTests/VBCSCompilerTests.csproj

```diff
@@ -20,6 +20,7 @@
     <ProjectReference Include="..\..\CSharp\csc\csc.csproj">
       <Aliases>csc</Aliases>
     </ProjectReference>
+    <ProjectReference Include="..\..\Test\Utilities\CSharp\CSharpCompilerTestUtilities.csproj" />
```

Comment:
Here is the layering of our utility projects

TestUtilities.csproj: helpers that are ideally language neutral and can run on any target framework 
CSharpCompilerTestUtilities / BasicCompilerTestUtilities: language specific compiler utility libraries 

VBCSCompiler is a mix of C# and VB tests so it's not inherently a violation for it to use either of the language specific compiler utilities. It is a bit odd in this specific case though since it's a generic touched files tests and it's using the C# specific test base. Let me see if that can be unwound. 



---
In reply to: [172261510](https://github.com/dotnet/roslyn/pull/25213#discussion_r172261510) [](ancestors = 172261510)